insights/parsers/uname.py

```diff
@@ -333,6 +333,9 @@ def parse_nvr(cls, nvr, data=None, arch=True):
             data = dict(cls.defaults)
 
         data['version'], data['release_arch'] = nvr.split('-', 1)
+        # Debug kernels have .debug appended to the end, so remove it before continuing.
+        if data['release_arch'].endswith('.debug'):
+            data['release_arch'] = '.'.join(data['release_arch'].split('.')[:-1])
```

Comment:
@ryan-blakley I think it makes sense to create a new attribute, maybe `debug` or `debug_kernel` and set it based on whether this is present.  Also I think you could use `.rsplit('.', 1)[0]` instead to avoid having to rejoin the parts back together.

Reply:
@bfahr Added the new attribute, and switched to rsplit.